blocks/editable/block.scss

```diff
@@ -0,0 +1,16 @@
+.drop-cap-true .blocks-editable__tinymce:not( :focus )  {
```

Comment:
Yeah, we can probably move this and remove the tinymce stuff. I though it was somewhat odd that the drop cap was in editable and not in text. I had to do an `ack` to find where the drop cap was. I think the idea behind it though is that there could be other fields that want a drop cap when using editable?

I am also not sure whether we want to change the styles around too much in this PR, and instead have this PR be focused more around the actual piping of styles in our webpack configuration. Then in subsequent PRs we can figure out where the styles should go. I know @jasmussen would have good ideas on how to break things up and configure them. If we start changing the styles now, we would have to change things on the text block as well, because currently the drop cap class names are not even added. We can still do it in this PR, but I would like to avoid merge conflicts 😄